src/main/scala/akka/persistence/jdbc/util/PluginVersionChecker.scala

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2016 Dennis Vriend
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package akka.persistence.jdbc.util
+
+object PluginVersionChecker {
+  def check() =
```

Comment:
`: Unit`

(should probably not be public in the first place, but maybe that is not cared about in this version of the plugin?)

Reply:
Let's protect it. I will change.

Reply:
@patriknw 

- [x] `:Unit` 
- [x] private API 
- [x] `@InternalApi`  
- [x] ported to master #344

Reply:
thanks